### ISSUE TITLE
Custom denied camera permission message

### ISSUE THREAD

**Issue by DDihanov** [journeyapps/zxing-android-embedded#488]
Any planned support for a custom deny camera permission message/callback? Because right now I have to manually check for the permission before launching the intent, so that I can handle the deny appropriately.

Referring to:

https://github.com/journeyapps/zxing-android-embedded/blob/aec45c4f10e3769d7449bcb7b82b5d09689512a4/zxing-android-embedded/src/com/journeyapps/barcodescanner/CaptureManager.java#L271-L272

**Comment by rkistner** [comment]
Currently you can override the message by specifying:

```xml
<string name="zxing_msg_camera_framework_bug">Your custom message.</string>
```

Unfortunately, this would override it for all errors.

Pull requests to improve this behaviour would be welcome.

**Comment by anipp100** [comment]
I added a PR which adds the possibility to set a custom message and also add the possibility to remove the dialog for the missing camera permission. In both cases the activity result Intent is set to cancelled and a boolean intent MISSING_CAMERA_PERMISSION is set which the user of the library can react on if using the library.

**Comment by rkistner** [comment]
Released in 4.1.0.
